README.md

```diff
@@ -0,0 +1,111 @@
+Alive Language Server
+=====================
+
+This is the language server for [Alive: The Average Lisp VSCode Environment](https://github.com/nobody-famous/alive).
+
+## Running the server
+
+For newcomers it could be difficult the start with Common LISP specially if the tech stack is not known yet. This instructions helps to have the server up and running as easy as possible. Steps are:
```

Comment:
"…difficult the start…" -> to start with CL" ?

"this instructions…" -> these instructions will help you get a server up and running as easily as possible. The required steps are: